Source/WebCore/page/EventHandler.cpp

```diff
@@ -1441,6 +1441,8 @@ bool EventHandler::scrollRecursively(ScrollDirection direction, ScrollGranularit
     if (!parent)
         return false;
     RefPtr localParent = dynamicDowncast<LocalFrame>(parent.get());
+    if (!localParent)
```

Comment:
This one should be easy to test, with a `scrollIntoView()` in a remote subframe.

Reply:
That doesn't seem to hit it.  Not sure why.

Reply:
This is only called from executeScrollLineUp, executeScrollLineDown, and WebPage::scroll